### DISCUSSION TITLE
Getting a dependency error

### DISCUSSION THREAD

**Discussion by richardryangarcia** [wevm/viem#127] [Question] [answered]
Im trying to using Viem in a very basic node / typescript application. When running `npx tsc` I get the following error message:

node_modules/.pnpm/viem@0.1.0_typescript@4.9.5/node_modules/viem/dist/chain-819c7ff7.d.ts:3:29 - error TS2307: Cannot find module 'ethers@6/types/providers' or its corresponding type declarations.

import { TopicFilter } from 'ethers@6/types/providers';

Found 1 error in node_modules/.pnpm/viem@0.1.0_typescript@4.9.5/node_modules/viem/dist/chain-819c7ff7.d.ts:3

Any ideas what could be causing this? using typescript 4.9.5 and ethers 6.0.8

Thanks for the help yal!

**Comment by jxom** [comment, marked as answer]
Any chance you could upload a minimal repository reproducing the issue?

UPDATE: Fixed in #128.

**Reply by tmm** [reply]
Fixed in #128

**Reply by richardryangarcia** [reply]
awesome thank you! will keep an eye out for when this gets merged in. 

here is a small ref repo i put up to recreate the issue https://github.com/richardryangarcia/bookish-octo-palm-tree

**Reply by jxom** [reply]
`viem@0.1.1` should be released in a few mins!

**Reply by richardryangarcia** [reply]
Just updated the package version and tried it and yes the issue is fixed. Thank you!
